test/TBD/installapi-flag.swift

```diff
@@ -1,18 +0,0 @@
-// REQUIRES: VENDOR=apple 
-// RUN: %empty-directory(%t)
-
-// 1. Emit two TBDs, one with -tbd-is-installapi set, and one without
-
-// RUN: %target-swift-frontend -typecheck %s -tbd-is-installapi -emit-tbd -emit-tbd-path %t/flag-provided.tbd
```

Comment:
Should the `-tbd-is-installapi` flag also be deprecated and eventually removed?

Reply:
Yes. 
> We should likely remove the command line options for this too but that might need to be staged.
Upstream now has support for tbd-v5 which is in json but also holds more attributes that need to be recorded.
For swift, I think it's just capturing the segment of which symbols will be defined in. 
 I'll do more thorough update then.